src/components/DisplayList.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+
+export default function DisplayList(props: { list: Array<any> }) {
```

Comment:
Is `Array<any>` actually an array of `any` type or an array of to-do items?

Reply:
Well I guess it is an `Array<string>` now but yes it is array of to-do items. Will update